internal/osbuild1/result.go

```diff
@@ -123,3 +126,52 @@ func (cr *Result) Write(writer io.Writer) error {
 
 	return nil
 }
+
+func (cr *Result) UnmarshalJSON(data []byte) error {
```

Comment:
This is obviously completely bonkers, but in a very impressive way. I think it hits the right balance between getting something useful and something we can use now. Let's go with this, and refactor once we have settled on the final format and have the luxury of more time.

Reply:
This definitely needs a bit of discussion.  We could define the output schema for osbuild and rely on it, or we could keep it flexible not bother processing it on the receiving end in composer.  I think @gicmo told me he'd prefer it wasn't parsed and was assumed completely opaque, but I might be misremembering.

Reply:
I'm fine with keeping this as-is for now, what do you think @gicmo? I agree that keeping it opaque might be nicer and all we really need (we just dump it as logs anyway), but we do anyway parse the result in some cases (koji), so I'm not sure we gain anything by not parsing it?